### PR TITLE
hello-rails-react:set-up

### PR DESCRIPTION
**Pull request by Sonickmumba**
In this pull request, I

- Created a new Rails app called 'hello-rails-react'.
        - Use the command `rails new -j webpack my_new_rails_app` as described in the previous lesson's tutorial.
 - Verified that the `jsbundling-rails` gem is added to your Gemfile.
 - Initialized your project with Git.
 - Installed needed node dependencies as described in the previous lesson's tutorial.
 - Created a `babel.config.js` in the root folder of your project as described in the previous lesson's tutorial.
 - Updated `webpack.config.js` to include the babel loader as described in the previous lesson's tutorial.
 - Made sure that your project has a Postgres database set up. 
 - Created a table for storing your messages and create 5 different greetings. 
 - Created an API endpoint that selects a random greeting from your table (you will need a controller with an action and Rails route).
 - Created a static view that will be the root of your app.

---------

- Created the `App` component with react-router and render it in your static view.
- Created the `Greeting` component that will display a greeting. Set it up as a route in your App component.
- Created a store, an action, and a reducer that will connect to your API endpoint to get the random greeting.
- Displayed the random greeting in your `Greeting` component.

### CODE REVIEW

File: app/javascript/application.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+//   return (
+//   <h1>
+//     Hello World! Sonick
+//   </h1>);
+// }
+
+// ReactDOM.render(
+//   <App />,
+//   document.getElementById('root'),
+// );
```
Comment (Whoistolu) [inline app/javascript/application.js]: - [ ] [OPTIONAL] Kindly remove all unused code in order for you to have a clean code base. 👍🏽

File: app/views/root/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<%# <h1>Root#index</h1>
+<p>Find me in app/views/root/index.html.erb</p> %>
```
Comment (Whoistolu) [inline app/views/root/index.html.erb]: - [ ] [OPTIONAL] Let's also remove all unused lines of code here as well so you can have a clean code base.

**Review by Whoistolu — APPROVED** [review]
### Hi @Sonickmumba 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — COMMENTED** [review]
Hi @Sonickmumba,

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**